### PR TITLE
Infrastructure: Update packages with npm audit fix --force

### PR DESCRIPTION
**Pull request by zcorpan**
Possibly this should be rebased after merging the existing [`dependencies` PRs](https://github.com/w3c/aria-practices/pulls?q=is%3Apr+is%3Aopen+label%3Adependencies) so it's not introducing new conflicts with those PRs.

cc @jesdaigle

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "htmlparser2": "^6.0.0",
     "husky": "^4.3.8",
     "lint-staged": "^10.5.3",
-    "npm-merge-driver": "^2.3.6",
+    "npm-merge-driver": "^0.0.0",
```
Comment (nschonni) [inline package.json:51]: Looks like something went wrong here

Comment (zcorpan) [reply package.json:51]: Yeah I found this curious as well. Thanks for catching it :)

Comment (zcorpan) [reply package.json]: Fixed in 1b9394d00b5f31a2fd4ef7d48dbd7d164d1f20a8

**Comment by zcorpan** [comment]
Currently 2 [`dependencies` PRs](https://github.com/w3c/aria-practices/pulls?q=is%3Apr+is%3Aopen+label%3Adependencies) open. When they are merged: if this one doesn't conflict, I think it should be ready for review and merge also.

File: package-lock.json [hunk 2]
```diff
@@ -4699,20 +4699,17 @@
       "dependencies": {
         "ansi-regex": {
           "version": "2.1.1",
-          "resolved": false,
-          "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
+          "bundled": true,
```
Comment (nschonni) [inline package-lock.json:4702]: Don't think I've seen this attribute before. What version of NPM are you running?

Comment (zcorpan) [reply package-lock.json:13526]: Hmm, I was apparently on 6.14.11. Updated to 7.5.6. I still see "bundled" in package-lock.json, though.

File: package-lock.json [hunk 3]
```diff
@@ -1,8 +1,8832 @@
 {
   "name": "aria-practices",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (nschonni) [inline package-lock.json:4]: NPM 7 re-writes the entire file with the new lockfile version

Comment (zcorpan) [reply package-lock.json:4]: Is this good or bad?

Comment (nschonni) [reply package-lock.json:4]: Not sure, there were some incompatibilities with Dependabot at one time, but I'm not sure if that's still the case

**Comment by zcorpan** [comment]
Resolved conflicts.

**Comment by mcking65** [comment]
Unsure of how to review this :( or when to merge.

I think I'll just rely on all of you ... @zcorpan, @jesdaigle, and @nschonni.

**Comment by nschonni** [comment]
I'd suggest closing this one till #1789 is resolved and skip the lockfile upgrade for now

**Comment by nschonni** [comment]
@zcorpan is this replaced by https://github.com/w3c/aria-practices/pull/2108

**Comment by zcorpan** [comment]
Yeah, let's close this. Thanks.

**Pull request closed without merging**